project-api/project-api-test/src/test/java/io/ballerina/projects/test/TestBuildProject.java

```diff
@@ -351,6 +353,36 @@ public void testJBallerinaBackend() {
         Assert.assertEquals(platformLibraries.size(), 4);
     }
 
+    @Test(description = "tests langlib files are not included in the jar file collection for executing the test")
+    public void testLanglibFilesAreNotIncluded() {
+        Path projectPath = tempResourceDir.resolve("myproject_with_lang_import");
+
+        // 1) Initialize the project instance
+        BuildOptions buildOptions = BuildOptions.builder().setSkipTests(false).build();
+        BuildProject project = loadBuildProject(projectPath, buildOptions);
+        // 2) Compile the current package
```

Comment:
Let's have a space before this comment as well to be consistant

Reply:
Done in dc5fa41554c2ad064a315787579aa292d971032b commit.